app/main/actions/exec.js

```diff
@@ -1,8 +1,8 @@
 const { fork } = require('child_process');
 
 module.exports = (message, arg) => {
-  if (arg) {
-    fork(arg, {
+  if (arg.script) {
```

Comment:
Although I agree with supporting backwards compatibility, I think that passing objects as an argument is preferred? And (assuming) there aren't that many plugins out and acknowledging this is still fairly early in development I would simply recommend updating the API to support the argument structure